### ISSUE TITLE
Error in building AveragePooling Layer

### ISSUE THREAD

**Issue by Aashish3970** [NeuromorphicProcessorProject/snn_toolbox#71]
I am trying to convert keras leNet model to SNN. I make the config file accordingly and get the error message as follows.

![image](https://user-images.githubusercontent.com/54250712/97525314-7bde5080-1974-11eb-85d8-92f5c74dde35.png)

This is my model :+1: 
![image](https://user-images.githubusercontent.com/54250712/97525171-3d489600-1974-11eb-84ff-524db4f03317.png)

**Comment by rbodo** [comment]
I cannot help you with this error. I've been able to run the Brian2 example script with various network configurations; it seems you have discovered a corner case where the number of connections seems to exceed a certain limit on the side of Brian2. You may want to adjust your layer sizes to try to avoid this limit.

By the way, the relu activation must come after a batch normalization layer. Otherwise the parser cannot fuse the BN layer with the preceeding conv layer.
